x/ibc/07-tendermint/misbehaviour.go

```diff
@@ -69,22 +75,9 @@ func checkMisbehaviour(
 		)
 	}
 
-	// Evidence is within the trusting period. ValidatorSet must have 2/3 similarity with trusted FromValidatorSet
-	// check that the validator sets on both headers are valid given the last trusted validatorset
-	// less than or equal to evidence height
-	if err := evidence.FromValidatorSet.VerifyFutureCommit(
-		evidence.Header1.ValidatorSet, evidence.ChainID,
-		evidence.Header1.Commit.BlockID, evidence.Header1.Height, evidence.Header1.Commit,
-	); err != nil {
-		return fmt.Errorf("validator set in header 1 has too much change from last known validator set: %v", err)
-	}
-
-	if err := evidence.FromValidatorSet.VerifyFutureCommit(
-		evidence.Header2.ValidatorSet, evidence.ChainID,
-		evidence.Header2.Commit.BlockID, evidence.Header2.Height, evidence.Header2.Commit,
-	); err != nil {
-		return fmt.Errorf("validator set in header 2 has too much change from last known validator set: %v", err)
-	}
-
-	return nil
+	return lite.Verify(
```

Comment:
lite.Verify function should be used to check if we can trust h2 if we trust h1, not an verify an evidence